modules/common/react/lib/utils/changeToStaticClass.ts

```diff
@@ -0,0 +1,28 @@
+import {CSSObject} from '@emotion/core';
+
+/**
+ * Allows the ability to force a state by a pseudo selector by changing the selector into a class creating a new styled object with the class names
+ * @param shouldChangePseudoSelectors If true, it will replace pseudo selectors from the styled object and change them to classes
+ * @param obj The styled object passed to the function to replace any pseudo selectors with a class name
+ */
+
+export const changeToStaticClass = (
+  shouldChangePseudoSelectors = false,
+  obj?: CSSObject
+): CSSObject | undefined => {
+  if (!shouldChangePseudoSelectors || !obj) {
+    return obj;
+  }
+
+  return Object.keys(obj).reduce((result, key) => {
+    const newKey = key
+      .replace(/^:/, '&:')
+      .replace(/:/g, '.')
+      .replace(/\.not/g, ':not')
+      .replace(/\.disabled/, ':disabled');
+    const value =
+      typeof obj[key] === 'object' ? changeToStaticClass(true, obj[key] as CSSObject) : obj[key];
+    const newObj = {...result, [newKey]: value};
+    return newObj;
+  }, {});
+};
```

Comment:
Looking for feedback @NicholasBoll @anicholls, Nicholas was mentioning that we can maybe use context when we refactor this. Currently with this implementation we need to add a hidden prop to the component to handle the `shouldChangePseudoSelectors`

Reply:
It is unfortunate the current implementation requires wrapping of each styled object/function. This is a bigger problem on button due to all the objects/functions defined in the `styled` factory function.

Reply:
@NicholasBoll @mannycarrera4 Wouldn't working it into https://github.com/Workday/canvas-kit/blob/master/modules/_labs/core/react/lib/theming/styled.ts address both of these concerns?

I realize we have the issue with infinitely nested style objects, but that's easily addressable in `styled.